assets/wizards/analytics/views/newspack-custom-events/index.js

```diff
@@ -0,0 +1,118 @@
+/* global newspack_analytics_wizard_data */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Button,
+	Grid,
+	Notice,
+	SectionHeader,
+	TextControl,
+	withWizardScreen,
+} from '../../../../components/src';
+
+/**
+ * Analytics Custom Events screen.
+ */
+class NewspackCustomEvents extends Component {
```

Comment:
Very much non-blocking and a stylistic choice, but worth noting that for newer components we try to prefer functional-style React components for consistency.

Reply:
yeah.. I just used the other components under Analytics as a starter and they were like this

Reply:
Yeah, those must have been old—when we have tons of free time we can refactor all of them. 😆